tests/integration/any_charm.py

```diff
@@ -8,9 +8,12 @@
 import pathlib
 import signal
 import subprocess
+from typing import Dict
 
-from any_charm_base import AnyCharmBase
-from ingress import IngressRequires
+from any_charm_base import AnyCharmBase  # type: ignore[import]
```

Comment:
I think these can be avoided by adding the dependencies to the `lint` job that `mypy` is looking for

Reply:
But this file is used inside the Any charm that has this dependency. any_charm_base is not used by the charm itself.

Reply:
Oh I see, it is something that would be added in the any charm context, so it wouldn't be possible to add this as a pip dependency

Reply:
Exactly :) I'll mark this as resolved but this can be reviewed in the story ISD-422.